### DISCUSSION TITLE
Cognito-Idp: I see 'code_delivery_details' is nil when signingUp.

### DISCUSSION THREAD

**Discussion by atharnoor2** [spulec/moto#7701] [Q&A] [answered]
I'm using moto via Docker and have setup Lambda, UserPool  and client. When I signup a new user in the pool, I see the `code_delivery_details` is always return as nil. Incase of ForgotPassword, it does turn `code_delivery_details` in response. Is this something not supported for signUp at the moment ?

**Comment by bblommers** [comment, marked as answer]
Hi @atharnoor2, this is indeed not yet supported yet - I've created a feature request for this here: #7708
